### PR TITLE
Consistently use the term "record" in AzureCloud

### PR DESCRIPTION
**Pull request by susam**
Replace any usage of the term "document" with "record" and get rid of the term "document" from the logs for security rules.

### CODE REVIEW

File: cloudmarker/clouds/azurecloud.py [hunk 1]
```diff
@@ -224,8 +224,7 @@ def _get_normalized_firewall_rules(nsg_doc, subscription_id):
                     _get_normalized_firewall_destination_ports(security_rule),
```
[the remark this answers is not in the document]
Comment (s-nirali) [reply cloudmarker/clouds/azurecloud.py]: Line 168 has `nsg_doc`

File: cloudmarker/clouds/azurecloud.py [hunk 2]
```diff
@@ -79,20 +79,20 @@ def read(self):
...
-                    _get_doc(storage_account_list, 'storage_account',
+            for record in itertools.chain(
+                    _get_record(vm_list, 'virtual_machine', subscription_id),
+                    _get_record(app_gw_list, 'app_gateway', subscription_id),
+                    _get_record(lb_iter, 'lb', subscription_id),
+                    _get_record(nic_list, 'nic', subscription_id),
+                    _get_record(nsg_list, 'nsg', subscription_id),
+                    _get_record(pubip_list, 'public_ip', subscription_id),
+                    _get_record(storage_account_list, 'storage_account',
                              subscription_id),
```
Comment (s-nirali) [inline cloudmarker/clouds/azurecloud.py]: Make checks fails here and at line `92` for indentation.

File: cloudmarker/clouds/azurecloud.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (susam) [reply cloudmarker/clouds/azurecloud.py]: Fixed [line 134](https://github.com/cloudmarker/cloudmarker/pull/74/files#diff-4fb06491b278fdc6193c4b1e9b8d0284R134).

Fixed [line 168](https://github.com/cloudmarker/cloudmarker/pull/74/files#diff-4fb06491b278fdc6193c4b1e9b8d0284R168).

File: cloudmarker/clouds/azurecloud.py (hunk 2 above)
Comment (susam) [reply cloudmarker/clouds/azurecloud.py]: Fixed [lines 90 and 92](https://github.com/cloudmarker/cloudmarker/pull/74/files#diff-4fb06491b278fdc6193c4b1e9b8d0284R89).

**Pull request merged**